integration/test-integration/pom.xml

```diff
@@ -220,6 +228,18 @@
             <artifactId>org.wso2.micro.gateway.enforcer</artifactId>
             <version>4.0.0-m10-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.am</groupId>
+            <artifactId>org.wso2.am.integration.common.test.utils</artifactId>
+            <version>4.0.0-SNAPSHOT</version>
```

Comment:
Shall we use something like 4.0.0-alpha

Reply:
+1

Reply:
4.0.0-alpha has not been released to nexus yet. Therefore, I will leave here as it is for the moment. Added a todo comment.